### ISSUE TITLE
line height doesn't work 4.6.0

### ISSUE THREAD

**Issue by cfconsultancy** [tinymce/tinymce#3624]
bug

in the content_css: body { line-height: 24px; } or p { line-height: 24px; } doesn't work

only if you set !important at the rule

Only in the latest 4.6.0 version

**Comment by czirkoszoltan** [comment]
This is caused by the rule in the content.css file which reads
```
.mce-content-body p,.mce-content-body div,.mce-content-body h1,.mce-content-body h2,.mce-content-body h3,.mce-content-body h4,.mce-content-body h5,.mce-content-body h6 {
 line-height:1.2em
}
```
This rule should be deleted completely, as it overwrites other stylesheets that are added via the `tinymce.init({content_css: ...})` option. The rules are just too strong (1 class + 1 tag name). A simple `body { line-height: 1.2em; }` would work instead, as it is weak, affects only one element, so very easy to override.

**Comment by czirkoszoltan** [comment]
This bug is already fixed, the newest version 4.7.9 does not have it. The bug can be closed.

However the skin creator web app at http://skin.tinymce.com/ still contains this bug, I guess it uses an old version (at least as of today).

**Comment by lnewson** [comment]
As noted above this was fixed between 4.6.0 and 4.7.9. I've also just tested the latest version of the [skin tool](http://skin.tiny.cloud/t4/) and it also no longer includes the problematic CSS. As such I'll go ahead and close this.
